src/annotations/AnnotationManager.php

```diff
@@ -299,6 +306,22 @@ protected function getAnnotations($class_name, $member_type = self::MEMBER_CLASS
                 }
             }
 
+            if ($this->_traitsSupported && $member_name) {
```

Comment:
Can you instead move whole IF statement inside IF where `$file` variable is being defined? Right now if annotations of a class internal class (e.g. `SplFileInfo`) are read, then `$file` variable won't be there and it might end up badly (e.g. notice/exception) when it's used.